core/templates/dev/head/services/QuestionsListService.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to fetch questions and returns questions to the
+ * questions list in editors.
+ */
+
+require('domain/question/QuestionBackendApiService.ts');
+require('services/ContextService.ts');
+
+oppia.constant(
+  'EVENT_QUESTION_SUMMARIES_INITIALIZED', 'questionSummariesInitialized');
```

Comment:
Move this to [services.constants.ts](https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/services/services.constants.ts)

Reply:
Done.